common/history/buildings/08_middle_east.txt

```diff
@@ -479,6 +479,20 @@
 				activate_production_methods={ "pm_anchorage" }
 			}
 		}
+		region_state:JAB={
+			create_building={
+				building="building_fishing_wharf"
```

Comment:
State Shammari Basra has 1 of building type Fishing Wharves and can only support 0, its level will be reduced.

Reply:
I hate to the limit of resources. Fixed